environments/react/v16.js

```diff
@@ -14,6 +14,10 @@ module.exports = {
     './v15.js',
```

Comment:
_.v15.js_ should also include the `react` plugin...

Reply:
v15 already has react plugin 😉 
https://github.com/strvcom/eslint-config-javascript/blob/master/environments/react/v15.js#L23

Reply:
Oooh I see! 👍